src/layers/basic.jl

```diff
@@ -85,10 +121,35 @@ _show_layers(io, layers::AbstractVector) = (print(io, "["); join(io, layers, ",
 """
     activations(c::Chain, input)
 
-Calculate the forward results of each layers in Chain `c` with `input` as model input.
+Calculates the forward results of each layers in Chain `c` with `input` as model input.
```

Comment:
```suggestion
Like calling a `Chain`, but saves the result of each layer as an output.
```
Might benefit from more workshopping on top of this.